samcli/local/docker/container.py

```diff
@@ -188,6 +186,12 @@ def create(self):
 
         if self.network_id == "host":
             kwargs["network_mode"] = self.network_id
+        else:
+            # only add when host network is not defined. 
+            # This issue stems from a change in docker-py
+            # https://github.com/docker/docker-py/commit/6da140e26c1cbe8e362b328b1c78d9c736f3a1a2
+            kwargs["ports"] = {self.RAPID_PORT_CONTAINER: (self._container_host_interface, self.rapid_port_host)}
```

Comment:
When `host` is provided, we need to set the port of rapid to `rapid_port_host` otherwise this container will run on port 8080, which is rapid's default.